mon-pix/app/styles/pages/_terms-of-service.scss

```diff
@@ -44,18 +41,21 @@
 
   &__description {
     margin-bottom: 40px;
+    font-size: 1rem;
```

Comment:
Ce css touche aux autres pages terms-of-service.
Pour checker le résultat des autres pages : 

- PE : faire le scénario PE pour accéder à la page des cgu PE
-  classique : se connecter avec sco.admin@example.net. Dans la console, ouvrir l'extension Ember et changer le mustValidateTermsOfService à true, puis cliquer sur un onglet au hasard